app/_src/gateway/production/monitoring/readiness-check.md

```diff
@@ -132,6 +132,6 @@ readinessProbe:
 
 For more information on Kong and related topics, check out the following resources:
 
-* [Kong Admin API Documentation](https://docs.konghq.com/gateway/latest/admin-api/)
-* [Get Started with Kong Ingress Controller](https://docs.konghq.com/kubernetes-ingress-controller/latest/deployment/)
```

Comment:
Should we add a CI check that disallows this URL (use `{{ site.links.web }}` instead)?

Reply:
I would say disallow in general. The instances that we actually need to link to `https://docs.konghq.com` and not use a relative URL are very rare (in fact, I currently can't think of one).

Reply:
We do it in k8s manifest URLs that are different per version

Reply:
I guess we could just have the error explain that - if you're linking to another doc page, use a relative link; if you have to use the direct link, use `{{ site.links.web }}` instead.

Reply:
Wonder if we could do this with Vale @Guaris ?

Reply:
Done https://github.com/Kong/docs.konghq.com/pull/5676